svelte-cloudinary/package.json

```diff
@@ -72,10 +72,22 @@
 	"types": "./dist/index.d.ts",
 	"type": "module",
 	"dependencies": {
-		"@cloudinary-util/url-loader": "3.17.0",
-		"@cloudinary-util/util": "2.1.0",
+		"@cloudinary-util/url-loader": "3.24.0-beta.10",
```

Comment:
I would just make sure this doesn't get merged in along with the other changes until it's released to stable